cherab/openadas/openadas.py

```diff
@@ -86,6 +86,21 @@ def recombination_rate(self, ion, charge):
 
         return RecombinationRate(data, extrapolate=self._permit_extrapolation)
 
+    def thermalchargeexchange_rate(self, ion, charge):
```

Comment:
How about relabeling to `thermal_cx_rate`? This is more in line with the others. For example, `beam_cx_pec` for the beam induced CX PEC.

Reply:
Also, the arguments are probably not broad enough. Shouldn't it be something like:
```
    def thermal_cx_rate(self, donor_ion, donor_charge, receiver_ion, receiver_charge):
```